### PR TITLE
Add support for periodic cleaning of the cache

### LINKED ISSUE #2: Improvement: Clean up expired keys
Currently we only clean up a cached value when it is next accessed so if we stop using a key, it may never get removed. Let's add a helper that can clean up expired keys.

We may need a way to track all the keys that have been set or a naming convention that allows us to find keys owned by the LocalCache to do this cleanup.

### PR DESCRIPTION
**Pull request by jbuckner**
Currently cache keys are only deleted when accessed. This adds a periodic cleaner to remove expired cache entries.

Closes #2

### CODE REVIEW

File: src/local-cache.ts [hunk 1]
```diff
@@ -58,9 +83,8 @@ export class LocalCache implements LocalCacheInterface {
     const namespacedKey = this.getNamespacedKey(options.key);
     try {
       await idbSet(namespacedKey, cacheEntry);
-    } catch {
-      // indexeddb may not be available (Firefox throws an error in Private mode)
-    }
+      // eslint-disable-next-line no-empty
+    } catch {} // indexeddb may not be available (Firefox throws an error in Private mode)
```
Comment (iisa) [inline src/local-cache.ts:87]: Q: what do we do when there isn't an indexeddb available - overall?

File: test/local-cache.test.ts [hunk 2]
```diff
@@ -139,4 +139,90 @@ describe('LocalCache', () => {
     expect(result.value).to.equal('bar');
     await localCache.delete('foo');
   });
+
+  describe('Cleaning', () => {
```
Comment (iisa) [inline test/local-cache.test.ts:143]: 👏

**Review by iisa — APPROVED** [review]
Looks good

File: src/local-cache.ts (hunk 1 above)
Comment (jbuckner) [reply src/local-cache.ts:87]: We just can't use the cache at all so every request/calculation has to be made fresh.

**Pull request merged**